web/gds-admin-ui/src/components/ResendEmail.js

```diff
@@ -0,0 +1,91 @@
+                return false
+            }
+            return true
+        }),
+    })
+);
+
+
+
+function ResendEmail({ toggle, modal, vaspName }) {
+
+    const { register, handleSubmit, formState: { errors }, control } = useForm({
+        shouldUnregister: true, resolver: schemaResolver,
+        defaultValues: {
+            rejection_reason: '',
+            email_type: null
+        }
+    });
+    const [, setResult] = React.useState("");
+    const emailType = useWatch({
+        control, name: "email_type"
+    })
+
+    const onSubmit = (data) => {
+        setResult(JSON.stringify(data))
+    };
+
+
+    return (
+        <div>
+            <Modal show={modal} dialogClassName="modal-right">
+                <Form noValidate onSubmit={handleSubmit(onSubmit)}>
+                    <Modal.Header onHide={toggle} closeButton>
+                        <h4 className="modal-title">Resend email</h4>
+                    </Modal.Header>
+                    <Modal.Body>
+                        <h5 className="mb-3">{vaspName}</h5>
+                        <p>What kind of email do you want to send ?</p>
+                        <div className="">
+                            <div>
+                                <Form.Label htmlFor="verifyContact" />
+                                <Form.Check {...register("email_type")} id="verifyContact" type="radio" value="verify contact" label="Resend verify contact email" required />
+
+                                <Form.Label htmlFor="review" />
+                                <Form.Check {...register("email_type")} id="review" type="radio" value="review request" label="Resend review request email" required />
+
+                                <Form.Label htmlFor="certificateDelivrey" />
+                                <Form.Check {...register("email_type")} id="certificateDelivrey" type="radio" value="certificate delivery" label="Resend certificate delivery email" required />
+
+                                <Form.Label htmlFor="rejection" />
+                                <Form.Check {...register("email_type")} id="rejection" type="radio" className="mb-1" value="rejection" label="Resend rejection email" required />
+
+                            </div>
+                            {emailType === "rejection" &&
+                                <Form.Group className="mb-3" controlId="validationRejectionReason">
+                                    <Form.Control {...register("rejection_reason")} as="textarea" rows={5} isInvalid={errors && errors["rejection_reason"] ? true : false} />
+                                    {errors && errors["rejection_reason"] ? (
+                                        <Form.Control.Feedback type="invalid">
+                                            {errors["rejection_reason"]['message']}
+                                        </Form.Control.Feedback>
+                                    ) : null}
+                                </Form.Group>
```

Comment:
I very much like the rendering behavior here that shows this text area only when the last radio button is selected! 

Is it possible to add help text to this field? "Please supply a rejection reason to resend the email"